backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -177,8 +239,59 @@ nav.menu {
   }
 }
 
+.admin-nav-menu li:not(.selected),
+.admin-nav-hidden .admin-nav-menu li.selected:hover {
+  position: relative;
+
+  // flyout nav
+  > ul {
+    position: absolute;
+    top: 0;
+    left: 100%;
+    width: $width-sidebar-flyout;
+    margin-left: 0;
+    background: $color-white;
+    box-shadow: 1px 0 4px 0 rgba(0, 0, 0, 0.1);
+    border-radius: 0 4px 4px 4px;
+    border: 1px solid $color-border;
+    @include caret($direction: left, $color-caret: $color-white);
+
+  &:before {
+    z-index: 1;
+    top: 1.5em;
+    pointer-events:none;
```

Comment:
missing space after `:`

Reply:
HoundCI should catch those 🤔

Reply:
@kennyadsl If I remember well, HoundCI notify you when the white space is missed, at the end of the file